### ISSUE TITLE
No open source model structure

### ISSUE THREAD

**Issue by zqnnn** [BertaBescos/EmptyCities#1]
Hi~I have read your paper,but I didn't see the model structure file in this project, will you open it?

**Comment by BertaBescos** [comment]
Hi! We are currently working on an improved and more extended version of this work. Once it will be finished (within a month or so) we plan to release everything on both torch and pytorch. Sorry for the inconvenience!

**Comment by zqnnn** [comment]
Looking forward to your open source code

**Comment by fiftywu** [comment]
How is the open source version of Pytorch going? Thank you.

**Comment by minastinisofking** [comment]
How is the open source version of Pytorch going? Thank you

**Comment by minastinisofking** [comment]
How is the open source version of Pytorch going? Thank you

**Comment by prashkmr** [comment]
Hi @BertaBescos  I do not see any finished code for model structure and training the model using different data.
Requesting you to share the full final code with the models with the Segmentation, mask, and the GAN module, we need to use it as a baseline for our paper. 
Thanks,
Prashant

**Comment by BertaBescos** [comment]
Hi Prashant,

We decided not to release the training code for the ICRA paper. However, we
have released the code of the journal version that will be soon published
in TRO. You can find the code in my GitHub repository EmptyCities_SLAM, and
a preliminary version of the paper in my website *https://bertabescos.github.io
<https://bertabescos.github.io>* under the link Dynamic-to-Static Image
Translation for Visual SLAM
<https://drive.google.com/file/d/1Lzjs30HAzc4oykY4IgPqQJKiLDnrm2td/view> in
the workshops section. I would be happy to know more about your research!

**Comment by prashkmr** [comment]
Thanks Berta. 
Your work is impressive and the illustrations are nice.
 Our work is on the lines of this work of yours, but in LiDAR SCAN space.
Pairing is a challenge for us even in the CARLA simulator as compared to
Images due to certain properties of SLAM algorithms for LiDAR Scans which
we try to overcome.
Good Luck.
